internal/value/value.go

```diff
@@ -386,6 +401,31 @@ func BoolValue(v bool) boolValue {
 
 type dateValue uint32
 
+func (v dateValue) CastTo(dst interface{}) error {
+	switch vv := dst.(type) {
+	case *string:
+		*vv = time.Unix(int64(v)*int64(time.Hour/time.Second)*24, 0).Format("2006-01-02")
+		return nil
+	case *[]byte:
```

Comment:
I suggest to not cast time to types other then string and time.Time. Because it can many ways to store time in binary form or int values (seconds, microseconds, nanoseconds).

String is intuivive time representation is time - for computer and string for human or universal (all our types can cast to string).